src/main/java/com/microsoft/azure/functions/worker/handler/WorkerInitRequestHandler.java

```diff
@@ -24,6 +24,7 @@ String execute(WorkerInitRequest request, WorkerInitResponse.Builder response) {
         response.putCapabilities("WorkerStatus", "WorkerStatus");
         response.putCapabilities("RpcHttpBodyOnly", "RpcHttpBodyOnly");
         response.putCapabilities("RpcHttpTriggerMetadataRemoved", "RpcHttpTriggerMetadataRemoved");
+        response.putCapabilities("HandlesWorkerTerminateMessage", "HandlesWorkerTerminateMessage");
```

Comment:
Should the value be "true", can we confirm with host team

Reply:
We only check if it's not null or empty, so value (as long as not empty) doesn't matter